rbi/core/kernel.rbi

```diff
@@ -361,11 +361,11 @@ module Kernel
 
   sig do
     params(
-        mod: Module,
+        arg0: Module,
     )
-    .returns(NilClass)
+    .returns(T.self_type)
   end
-  def extend(mod); end
+  def extend(*arg0); end
```

Comment:
Why is this changing in this PR?

Reply:
Since I'm leaving it in the tree, we now typecheck it and we have some tests that pass two params to `extend` (which is allowed by Ruby)